tests/ptf/fabric_test.py

```diff
@@ -150,6 +151,14 @@ def pkt_add_vlan(pkt, vlan_vid=10, vlan_pcp=0, dl_vlan_cfi=0):
            pkt[Ether].payload
 
 
+def pkt_add_inner_vlan(pkt, vlan_vid=10, vlan_pcp=0, dl_vlan_cfi=0):
```

Comment:
Out of curiosity: what's wrong with the previous way of adding the inner vlan, i.e. calling twice  pkt_add_vlan?

Reply:
To add 2 VLAN to a packet you can call 2 times the `pkt_add_vlan`. The problem is when you have a packet with a VLAN already set and you want to add an inner VLAN tag, in this case you have to use the `pkt_add_inner_vlan`.